test/iconGenerator.test.ts

```diff
@@ -2,34 +2,26 @@
 /* tslint:disable only-arrow-functions */
```

Comment:
I also like `arrow functions` but I will rather stick to [Mocha's recommendations](https://mochajs.org/#arrow-functions).